content/hardware/04.pro/boards/portenta-c33/essentials.md

```diff
@@ -10,7 +10,7 @@
         The Wi-Fi® library is designed to use the ESP32-C3-MINI-1U module, which allows your Arduino to connect to the Internet.    </EssentialElement>
 
 <EssentialElement link="https://www.arduino.cc/en/Reference/ArduinoBLE" title="Arduino BLE" type="library">
-    Bluetooth® 4.0 includes both traditional Bluetooth®, now labeled " Bluetooth® Classic", and the Bluetooth® Low Energy.
+    Bluetooth® 5.0 includes both traditional Bluetooth®, now labeled " Bluetooth® Classic", and Bluetooth® Low Energy.

```

Comment:
Good cath but I'm afraid you are wrong. Let me explain:

* Most of our recent boards are Bluetooth 5.0 by hardware
* But the Library Arduino BLE it still not compatible with Bluetooth 5.0.
Please only apply the removal of the "the"